86-get-indices-of-n-maximun.md

```diff
@@ -0,0 +1,18 @@
+# How to get indices of N maximum values in a NumPy array?
+
+NumPy proposes a way to get the index of the maximum value of an array via `np.argmax`.
+
+We can get the indices of N maximum values in a NumPy array using the below code:
+
+```python
+import numpy as np
+arr = np.array([1, 3, 2, 4, 5])
+arr.argsort()[-3:][::-1]
+array([4, 3, 1])
```

Comment:
what is np and arr?

Reply:
I am missing something to install?
![image](https://user-images.githubusercontent.com/12773753/126868783-50b66498-3cd7-45f3-832a-0d937fdaaa81.png)